### PR TITLE
Eliminate RobloxPlayerLauncher (API overhaul)

### PR DESCRIPTION
**Pull request by Thelolguy1**

### CODE REVIEW

File: dirs.go [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
+	Versions  string
 }
 
 var (
-	Dirs                  = defDirs()
-	DirMode   fs.FileMode = 0755
-	DirROMode fs.FileMode = 0664
-	FileMode  fs.FileMode = 0644
+	Dirs                = GetDirectories()
+	DirMode fs.FileMode = 0o755
```
Comment (Thelolguy1) [inline dirs.go:22]: typo?

Comment (apprehensions) [reply dirs.go:22]: Go has these for octal style literals, and this change was done by the `gofumpt` linter.

**Review by Thelolguy1 — COMMENTED** [review]
Reviewed, pending test

**Comment by Thelolguy1** [comment]
Not ready for merge:

Vinegar breaks in this procedure:

1. Launching player
2. Editing config
3. Launching studio (config appears to vanish?)

**Comment by Thelolguy1** [comment]
Working bits are:

Player installation + launching

**Pull request closed without merging**